stylesheets/commons/Panels.less

```diff
@@ -27,3 +27,71 @@ Author(s): FO-nTTaX
 .panel-box-body {
 	padding: 10px;
 }
+
+/* Temp, for testing purposes */
+
+.dota2-test {
+	.panel-box {
+		box-shadow: 0 0.0625rem 0.25rem 0 rgba( 0, 0, 0, 0.12 );
+		border: 0;
+		border-radius: 0.5rem;
+
+		&.is--collapsed {
+			.panel-box-heading {
+				border-radius: 0.5rem;
+
+				&-icon {
+					transform: rotate( 180deg );
+				}
+			}
+
+			.panel-box-body {
+				display: none;
+			}
+		}
+	}
+
+	.panel-box-heading {
+		display: flex;
+		align-items: center;
+		background-color: var( --clr-cinnabar-30 );
```

Comment:
This should probably be a theme specific class, and it probably should have a Bruinen fallback